src/status_im/contexts/shell/jump_to/pr_error_indicator.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.contexts.shell.jump-to.pr-error-indicator
+  (:require [quo.core :as quo]
+            [react-native.reanimated :as reanimated]
+            [react-native.safe-area :as safe-area]
+            [utils.re-frame :as rf]))
+
+(defn chat-open-error
+  []
+  (let [current-chat-id (rf/sub [:chats/current-chat-id])
+        view-id         (rf/sub [:view-id])]
+    (when (and (not (#{:chat :profile} view-id)) current-chat-id)
+      "Error: Chat is not closed. \n(messages will be marked read automatically).\nPlease create an issue with the steps that lead to this state.")))
+
+;; https://github.com/status-im/status-mobile/pull/18338
+(defn f-view
```

Comment:
better to make the f-view private and export as `view` that already handles the `[:f> f-view]` etc imo

Reply:
i.e
```
(defn view []
 [:f> f-view])
```
...
```
(when config/pr-error-indicator?
       [pr-error-indicator/view])
```

Reply:
I am little curious about this one.
Not sure how or if this affects, but should we avoid using another wrapper?
(Not sure performance effect, but probably we can check this wrapper in inspector (have to confirm))